src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp

```diff
@@ -449,19 +456,29 @@ namespace Opm {
     static std::vector< double > findKrgr( const TableManager* tm ) {
         const auto num_tables = tm->getTabdims().getNumSatTables();
         const auto& sgofTables = tm->getSgofTables();
+        const auto& slgofTables = tm->getSlgofTables();
         const auto& sgfnTables = tm->getSgfnTables();
 
-        const auto& famI = [&sgofTables]( int i ) {
+        const auto& famI_sgof = [&sgofTables]( int i ) {
             return sgofTables.getTable< SgofTable >( i ).getKrgColumn().front();
         };
 
+        const auto& famI_slgof = [&slgofTables]( int i ) {
+            return slgofTables.getTable< SlgofTable >( i ).getKrgColumn().back();
+        };
```

Comment:
This is in the spirit of the existing function, but in reality this isn't how you define `KRGR`.  That is a more involved calculation.  We'll pick that up later.